src/scr.c

```diff
@@ -767,6 +814,9 @@ static int scr_get_params()
   } else {
     scr_jobid = scr_env_jobid();
   }
+  if (scr_my_rank_world == 0) {
+     scr_dbg(1, "SCR_JOB_ID=%s", scr_jobid);
+  }
```

Comment:
Let's move this down just below the following block to come at line 832.  That block can set scr_jobid if it is NULL.